Directory.Build.props

```diff
@@ -26,8 +26,8 @@
   <PropertyGroup>
     <!-- For files to appear in the Visual Studio Solution explorer given we have conditional inclusion in some projects (IdWeb for instance)
          we need to have the higher framework, even if this produces a warning in the IDE -->
-    <TargetFrameworks>net6.0; net7.0; net8.0; net462; net472; netstandard2.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(TargetNet9)'== 'True'">$(TargetFrameworks); net9.0</TargetFrameworks>
+    <TargetFrameworks>net462; net472; netstandard2.0; net6.0; net7.0; net8.0; net9.0</TargetFrameworks>
```

Comment:
there was a reason for the order, it was because it wasn't picking up all the frameworks. So not sure if it needs to move back to how it was previously.

Reply:
Reverted.